srtp/srtp.c

```diff
@@ -774,7 +787,87 @@ static srtp_err_status_t srtp_kdf_clear(srtp_kdf_t *kdf)
     return srtp_err_status_ok;
 }
 
-#else  /* if OPENSSL_KDF */
+#elif defined(WOLFSSL) && defined(WOLFSSL_KDF)
```

Comment:
The OPENSSL_KDF is in practice never used ( at least not that I know off) and there has been some talk of removing it, are you sure you want to add wolfssl specific kdf functions?

Reply:
Customers needing FIPS 140 validated cryptography for SRTP need to use an implementation of SRTP KDF in the boundary.
OpenSSL doesn't support SRTP KDF unless you get a fork as I understand it and has therefore that code has less value.

Reply:
ok, have you verified that the tests pass with wolfssl kdf ? The one other time I have seen some one use an openssl kdf then it was broken.

Reply:
Yes, I have verified that the SRTP KDF in wolfSSL works in libsrtp.

Reply:
I had just quick thought that it was why the test fails, but I see now it was build issues not test failures